### PR TITLE
chore: drop support for license files (ee5.0)

### PR DESCRIPTION
**Pull request by thalesmg**
Related issue: EMQX-7247

EMQX ee5.0 is to drop support for both the old license format (x509) and for files containing the new format.

### CODE REVIEW

File: bin/emqx [hunk 1]
```diff
@@ -499,22 +499,16 @@ check_license() {
...
-            set +x
-            echoerr "License not found."
-            echoerr "Please specify one via EMQX_LICENSE__KEY or EMQX_LICENSE__FILE variables"
-            echoerr "or via license.key|file in emqx_enterprise.conf."
-            return 1
-        fi
+      set +x
+      echoerr "License not found."
+      echoerr "Please specify one via the EMQX_LICENSE__KEY variable"
+      echoerr "or via license.key in emqx_enterprise.conf."
```
Comment (zmstone) [inline bin/emqx:510]: We are in the middle of renaming:
```suggestion
      echoerr "or via license.key in emqx-enterprise.conf."
```

**Review by zmstone — APPROVED** [review]

**Comment by zmstone** [comment]
send to master branch?

File: lib-ee/emqx_license/src/emqx_license_schema.erl [hunk 2]
```diff
@@ -34,55 +32,13 @@ roots() ->
 
 fields(key_license) ->
     [
-        {type, #{
-            type => key,
-            required => true,
-            desc => ?DESC(license_type_field)
-        }},
         {key, #{
```
Comment (zmstone) [inline lib-ee/emqx_license/src/emqx_license_schema.erl:35]: Please move the default value from config file to here.
(a requirement from cloud-native team, for smooth upgrade from opensource edition to enterprise edition).

Comment (thalesmg) [reply lib-ee/emqx_license/src/emqx_license_schema.erl:35]: :heavy_check_mark:

File: bin/emqx (hunk 1 above)
Comment (thalesmg) [reply bin/emqx]: :heavy_check_mark:

**Comment by thalesmg** [comment]
Changed to #8961

**Pull request closed without merging**